api/workload/x509_client.go

```diff
@@ -1,11 +1,26 @@
 package workload
 
 import (
+	"context"
+	"errors"
+	"fmt"
 	"net"
+	"sync"
 	"time"
 
+	"github.com/andres-erbsen/clock"
 	"github.com/sirupsen/logrus"
+	"github.com/spiffe/spire/api/workload/dial"
 	"github.com/spiffe/spire/proto/api/workload"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/metadata"
+	"google.golang.org/grpc/status"
+)
+
+const (
+	DefaultTimeout    = 5 * time.Minute
```

Comment:
Not sure if we really need to export this. If we are going to export it, maybe it would be good to add a comment (that could reference the corresponding member in the X509ClientConfig struct).

Reply:
I'll unexport for now.